tests/test_create.py

```diff
@@ -2031,12 +2061,22 @@ def test_conda_pip_interop_pip_clobbers_conda(clear_package_cache: None):
     context.subdir not in ("linux-64", "osx-64", "win-32", "win-64", "linux-32"),
     reason="Skip unsupported platforms",
 )
-def test_conda_pip_interop_conda_editable_package(clear_package_cache: None):
+def test_conda_pip_interop_conda_editable_package(clear_package_cache: None, request):
```

Comment:
Some pip interoperability tests do pass, but this one requires locking the matching specs. I think we can make it work, but that's almost a new feature, so I'll start that separately in https://github.com/conda/conda-libmamba-solver/pull/324